mobile/lib/main.dart

```diff
@@ -212,6 +213,8 @@ Future<void> _init(bool isBackground, {String via = ''}) async {
     Computer.shared().turnOn(workersCount: 4).ignore();
     CryptoUtil.init();
 
+    IsolatedFfmpegService.init();
```

Comment:
Let's init it right before processing any video to avoid creating redundant isolate.